rir/src/interpreter/interp_data.h

```diff
@@ -387,17 +424,6 @@ typedef struct Function {
     unsigned markOpt : 1;
     unsigned spare : 28;
 
-    // TODO(mhyee): remove origin and next
```

Comment:
From what I understood, we keep them and have a 2d dispatch table, right?

Reply:
Correct.